ouroboros-consensus/src/Ouroboros/Consensus/Protocol/PBFT.hs

```diff
@@ -201,7 +201,10 @@ instance (PBftCrypto c, Typeable c) => OuroborosTag (PBft c) where
                                 , maybe (SlotNo 0) snd $ Seq.lookup (Seq.length chainState) chainState
                                 )
 
-      unless (blockSlot b > lastSlot)
+      -- FIXME confirm that non-strict inequality is ok in general.
```

Comment:
@nc6 can you verify this? (I have the feeling you have verified this already at my request in the last week, but it might have been something else.)